### DISCUSSION TITLE
Tagging gists

### DISCUSSION THREAD

**Discussion by dgarciabriseno** [community/community#14720] [Other Feature Feedback, Questions, & Ideas]
What do you guys think of having a way to tag gists rather than using the hash?

I find myself writing short scripts that I need to share between different systems. Whether it's a function in my .bash_profile or a quick command to find certain files. I work on about 5 different environments, (My work PC, personal linux PC, personal Mac, Work prod server, work dev server) and something I wrote on one system isn't available on the others until I manually copy it. I've started using gists to do this. Push a gist, pull it on another computer and that helps with transferring, but it would be even more convenient if my gists could have tags.

With tagged gists, instead of listing all your gists and finding the one you need, or logging into the browser to find and copy your gists, you would be able to simply name snippets as needed, then pull them somewhere else by referencing a tag like `dgarciabriseno/my-cool-gist`

This would make sharing public gists better too since again it's easier to reference a named tag than a hash. And we could share our tags verbally with friends and colleagues instead of digging for the hashes ourselves.

**Comment by HelderMagalhaes** [comment]
I'm a fan of the tagging idea, similar to already existing GitHub repositories.

> This would make sharing public gists better too since again it's easier to reference a named tag than a hash.

To be highlighted that a few (up to 6 repos and gists) can be pinned in the user profile. Far from a workaround or an actual fix but a currently available way to share a few gists (~"static", no appropriate for collections or such).
